react-native/react/actions/login2.js

```diff
@@ -277,25 +268,234 @@ export function logout () {
   }
 }
 
-export function setDeviceName (name) {
-  return function (dispatch) {
-    // TODO integrate
-    setTimeout(() => {
-      const error = false
-
-      if (error) {
+// Show a user/pass screen, call cb(user, passphrase) when done
```

Comment:
I mean the comment says `cb` is called with `user, passphrase` but it's actually called with no arguments: `cb()`